### ISSUE TITLE
Version scheme incorrectly using "beta"

### ISSUE THREAD

**Issue by alerque** [hyprwm/Hyprland#1944]
Please drop the `beta` suffix from future tagging. The use of the "beta" label in the tag here is not denoting the status of a *specific version*, it is being used to denote the overall status of the *project*. This is evidenced by the fact that the next version after `v0.22.0beta` or similar is never `v0.22.0`, but always another bump like `v0.23.0beta`. In other words these are not beta cuts of a specific tag that isn't marked yet, they **are** the release of that tag—with "beta" tacked on the end so people don't take the project too seriously.

Following either 0ver or semver specs as you prefer is fine, but the current scheme is weird and obnoxious for packagers. The leading zero already suggests the project is in a development mode where breaking changes can happen in any/all releases. Given the volume of uptake already in the wild the "don't use this as your daily driver" ship has sailed.

**Comment by vaxerski** [comment]
fair point, I don't see many reasons to keep the beta in anymore anyways tbh

**Comment by alerque** [comment]
Of course I *could* but that would confuse end user ... for example I packaged this for official Arch Linux repositories and if I'd removed the beta from the string we'd have had people wondering where I got the official release when the upstream project tag is different. As a general rule confusing people and generally doing different things than upstream marks is a not a great plan. Upstream projects doing things in a standardized way (like semver or 0ver or whatever) is the right way to make this easy on everybody.
